packages/core/src/layout/HeaderTabs/HeaderTabs.tsx

```diff
@@ -38,13 +38,13 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-export type Tab = {
+export type objectTab = {
```

Comment:
Types should always be uppercase. Also this type is actually exported so it's part of the contract - can we keep this  the way it was, and adapt elsewhere instead?